electrum/synchronizer.py

```diff
@@ -50,11 +50,29 @@ def history_status(h):
     if not h:
         return None
     status = ''
-    for tx_hash, height, *__ in h:
-        status += tx_hash + ':%d:' % height
+    for tx_hash, height, tx_type in h:
+        status += f'{tx_hash}:{height:d}:{tx_type:s}:'
     return bh2u(hashlib.sha256(status.encode('ascii')).digest())
 
 
+def filer_out_double_atxid_from_history(raw_history) -> list:
```

Comment:
typo: `filer`
btw: i would name it `get_unique_atxid_form_history` 
filter sounds more like operation `in place`

Reply:
done :)